package.json

```diff
@@ -30,8 +30,8 @@
     "url": "https://github.com/andig/pimatic-fritz"
   },
   "dependencies": {
-    "smartfritz-promise": "^0.4.1",
```

Comment:
Not, really. I changed it to 0.5.0 and then reverted it to 0.4.2. Anyhow, does it hurt to accept the change?